src/ReleaseHistory.md

```diff
@@ -1,5 +1,8 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+* FEATURE: SDK BREAKING CHANGE: Upgrade from .NET Framework 4.5 to .NET Framework 4.5.2 [#2135](https://github.com/microsoft/sarif-sdk/pull/2135)
```

Comment:
>FEATURE: SDK BREAKING CHANGE [](start = 2, length = 28)

We usually call this kind of change 'PACKAGE BREAKING'. i.e., it is a compatibility break at the package level. It's not a great name. :) But that's the pattern.

Reply:
just updated!

---
In reply to: [513101256](https://github.com/microsoft/sarif-sdk/pull/2135#discussion_r513101256) [](ancestors = 513101256)